lib/resources/styles.css

```diff
@@ -562,6 +562,17 @@ button {
     margin-right: 24px;
   }
 
+  #overlay-under-drawer.active {
```

Comment:
What properties does `#overlay-under-drawer` have if not active? It looks like a 0 width height div. Perhaps have it be display: none?